metrics_test.go

```diff
@@ -15,3 +15,29 @@ limitations under the License.
 */
 
 package main_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	main "github.com/RedHatInsights/ccx-notification-writer"
+)
+
+// TestDropTableStatement function checks the helper function
```

Comment:
just update the comment and it's good to go =)

Reply:
ah leftover :)

Reply:
Fixed